### ISSUE TITLE
New window opens all the pinned tabs on both workspaces?

### ISSUE THREAD

**Issue by darkvergus** [zen-browser/desktop#2839]
### Captchas

- [X] I have read the instructions.
- [X] I have searched existing issues and avoided creating duplicates.
- [X] I am not filing an enhancement request.

### What happened?

Easily reproducible, just open a new window with pins and the new window will contain those pins.

https://github.com/user-attachments/assets/47b2b80d-8e8c-4ae9-8a52-84b38cf407a5

### Reproducible?

- [X] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

1.0.1-a.18

### What platform are you seeing the problem on?

Windows

**Comment by salamijason** [comment]
Was able to recreate in a.18 but not t.18 but apparently author also got it in t.18

**Comment by JSGRD22** [comment]
Same here on flatpak, linux. Opening a new window opens all the tabs in all workspaces in 1 place. In addition, for me, container tabs, whose labels should be hidden in their respective default workspaces, all show their labels. Furthermore, workspaces are nonexistent in the new window and the workspace selection bar is completely empty, which could be the root of the problem.

**Comment by darkvergus** [comment]
Also happens in private mode with info from Fushiguro in discord

https://github.com/user-attachments/assets/deeb314c-0266-4ba4-bb20-1bca86258327

**Comment by universish** [comment]
videos cannot be played.

**Comment by darkvergus** [comment]
Its working fine on my end

**Comment by universish** [comment]
It's fixed. I can play it now.

**Comment by universish** [comment]
This is a feature that I want. As far as I'm concerned, it's not a problem. It's more of a useful feature. Isn't that the purpose of stabilization.
If you don't want to see pinned tabs in the new window, run the new window with a new workspace.

**Comment by universish** [comment]
I didn't understand anything from the video. Can you explain the problem in more detail and more clearly? 
Please

**Comment by universish** [comment]
> Same here on flatpak, linux. Opening a new window opens all the tabs in all workspaces in 1 place.

I don't have this problem with either of my OSes.
I use fedora 41 and windows 11.
I use appimage on fedora 41. 
in windows 11 I use the installable version.
I don't have this problem in both.
What is your operating system?

 And which version do you use (can other people who have this problem answer?) ?

**Comment by salamijason** [comment]
I believe this has been fixed? Looks like it's been resolved for the author. @JSGRD22, are you still getting this issue on the latest ver? (.19 as of this comment)

**Comment by darkvergus** [comment]
No it hasn't i still have the issue

**Comment by darkvergus** [comment]
I even have it on twilight

**Comment by salamijason** [comment]
mb! 😅 confused ur response [here](https://github.com/zen-browser/desktop/issues/2839#issuecomment-2468626784).

**Comment by salamijason** [comment]
Will try verifying on our end on linux.

**Comment by RayZ3R0** [comment]
Yep I have that bug too. I get pinned tabs from other workspaces into my newly opened window even though I did not have those pinned tabs in the workspace I was in. The new windows does open in correct workspace but with wrong pinned tabs. I've found that it gets kinda fixed if you switch back and forth on the workspaces, but not always

**Comment by salamijason** [comment]
Happens on both windows and linux a19 💀

**Comment by jkcdarunday** [comment]
Still happening on a22. Also messes with windows opened by extensions like Metamask.

**Comment by JSGRD22** [comment]
Has this been resolved?

**Comment by mr-cheffy** [comment]
Yeah
